.eslintrc

```diff
@@ -1,24 +1,19 @@
 {
-    "parser": "babel-eslint",
-    "extends": [
-      "prettier"
-    ],
-    "plugins": [
-      "prettier"
-    ],
-    "parserOptions": {
-      "ecmaVersion": 2016,
-      "sourceType": "module"
-    },
-    "rules": {
-      "no-unused-vars": "warn",
-      "prettier/prettier": ["error", {
-        "singleQuote": true,
-        "semi": false
-      }]
-    },
-    "env": {
-      "es6": true,
-      "node": true
-    }
-}
\ No newline at end of file
+  "extends": ["plugin:prettier/recommended"],	
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "no-unused-vars": "error",
+    "prettier/prettier": ["error", {
+      "printWidth": 120,
+      "singleQuote": true,
+      "semi": false
+    }],
+    "curly": "error"
+  },
+  "env": {
+    "es6": true,
+    "node": true
+  }
+}
```

Comment:
This is different from the .eslintrc file in vehicle-configs, which I assume is the repo with the most up-to-date configuration files.

Reply:
This is the one from API, because this project is still in JS.